src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+  let res = 0;
+
+  if (days < 3) {
+    res = sum;
+  }
+
+  if (days >= 3 && days < 7) {
+    res = sum - 20;
+  }
+
+  if (days >= 7) {
+    res = sum - 50;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return sum,
  }

  if (days < 7) {
   return sum - 20;
  }

  return = sum - 50;
```
Из if можно убрать дополнительные условия. Ты в первом if уже проверила что `days < 3`, значит если начал выполняться второй if  - то days уже точно больше 3